.github/workflows/publish.yml

```
@@ -0,0 +1,40 @@
+name: Publish Assets
+run-name: Publish assets to npm
```

```suggestion
name: Publish Teraslice NPM Packages and Docker Image
```

Changing the name to get rid of assets and removing `run-name` entirely.